src/coreComponents/mesh/WellElementSubRegion.cpp

```diff
@@ -115,29 +115,27 @@ void collectLocalAndBoundaryNodes( InternalWellGenerator const & wellGeometry,
 }
 
 /**
- * @brief Check if "location" is contained in reservoir element ei
- * @param[in] subRegion the subRegion of reservoir element ei
- * @param[in] ei the index of the reservoir element
- * @return true if "location" is contained in reservoir element ei, false otherwise
+ * @brief Check if "location" is contained in reservoir element
+ * @param[in] referencePosition the coordinates of the nodes
+ * @param[in] elemsToFaces the map from element to faces
+ * @param[in] facesToNodes the map from face to nodes
+ * @param[in] elemCenter the coordinates of the element center
+ * @param[in] location the target coordinates
+ * @return true if "location" is contained in the reservoir element, false otherwise
  */
-bool isPointInsideElement( NodeManager const & nodeManager,
-                           R1Tensor const & location,
-                           CellElementSubRegion const & subRegion,
-                           localIndex ei )
+bool isPointInsideElement( arrayView2d< real64 const, nodes::REFERENCE_POSITION_USD > const referencePosition,
```

Comment:
Hasn't this function become an alias like
```c++
bool isPointInsideElement( arrayView2d< real64 const, nodes::REFERENCE_POSITION_USD > const referencePosition,
                           arraySlice1d< localIndex const > const elemsToFaces,
                           ArrayOfArraysView< localIndex const > const facesToNodes,
                           real64 const (&elemCenter)[3],
                           real64 const (&location)[3] )
{
  return computationalGeometry::isPointInsidePolyhedron( referencePosition,
                                                         elemsToFaces,
                                                         facesToNodes,
                                                         elemCenter,
                                                         location );
}
```
Consider simply removing it?

Reply:
Yes, good idea, thanks!